### DISCUSSION TITLE
Unauthorized to visit Profile page, even after successfully login

### DISCUSSION THREAD

**Discussion by tigerinus** [requarks/wiki#5476] [Error / Bug Report] [answered]
Hello,

It's first time I setup Wiki.js on my own server yesterday and everything went well.

This morning, I am able to successfully login,

![image](https://user-images.githubusercontent.com/7172560/179313732-cbdfb70b-4bc6-4c76-ac95-cb582b715e12.png)

![image](https://user-images.githubusercontent.com/7172560/179313819-c3c7f7b6-44f6-423e-b521-a447a895ab3c.png)

However, when I try to visit Profile page, I always get Unauthorized error

![image](https://user-images.githubusercontent.com/7172560/179313273-39d5e85c-859f-491e-97fb-65d02ed7a93e.png)

What is a good starting point for troubleshooting? 

(I did clear all browser cache but it still repros)

Thanks.

# UPDATE - 1

This issue is gone after rebooting server. 

# UPDATE - 2

This issue came back. Rebooting server does not help.

**Comment by NGPixel** [comment, marked as answer]
I experienced this very strange bug a few weeks ago, but a simple restart solved it and have not been able to reproduce it since then. So unfortunately, I don't have any solution for you at the moment.

**Reply by tigerinus** [reply]
(Un)fortunately, this issue is gone for some reason I don't know. I did the same thing as you did, which is just restart few times.

**Reply by CKylinMC** [reply]
Facing same problem and restart solved it, thanks for your tip!
(Docker, Wiki.js 2.5.298, behind a reverse proxy)

**Comment by Facinorous-420** [comment]
I am getting this issue on fresh installs on the docker instance.
I notice it does not persist in the local ip, only when I am using my domain (which is using cloudflare for DNS, and NGINX as reverse proxy)

I get it both using the linuxserver container, as well as the official container.
This error is rather rough for me, as on a fresh install I can not log in as an administrator to even create a first page.
Auth in general doesnt work, it says it logs in successfully but gives an unauthorized page when trying to actually access pages meant for that user.. and my wiki is meant to be one locked behind a login, so I will be continuing to try to get to the bottom of why this is happening.

**Reply by tigerinus** [reply]
Sounds like your NGINX or some layer in-between wiki and ur browser might be caching the API calls...

**Reply by Facinorous-420** [reply]
I do not think it is cloudflare, as I can turn it to DNS only and it still happens. It must be my NGINX.

I find it strange as I had wikijs working perfectly fine for close to 2 years then I went to move my wiki onto another VM in the same host, and I started to get this. This is my NGINX config, I dont see anything inherently wrong with it at first glance and this is the one Ive been using for close to 2 years now for wikijs. https://p.bss.nz/mifuhujila.nginx
